build_projects/dotnet-cli-build/DotNetNew.cs

```diff
@@ -12,16 +12,18 @@ protected override string Command
 
         protected override string Args
         {
-            get { return $"{GetTemplateType()}"; }
+            get { return $"{GetTemplateType()} {TemplateArgs}"; }
         }
 
         public string TemplateType { get; set; }
 
+        public string TemplateArgs { get; set; }
+
         private string GetTemplateType()
```

Comment:
👍 Fixed in the latest commit https://github.com/dotnet/cli/pull/5430/commits/17279f60d0e174df901d39fbbc5c009a09ab04ea